ShopAppTests/App/Account/Providers/Mocks/OrdersListTableProviderMock.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  OrdersListTableProviderMock.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 2/21/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import UIKit
+
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class OrdersListTableProviderMock: NSObject, OrdersListTableProviderDelegate, CheckoutCartTableViewCellDelegate {
```

Comment:
Use `OrdersListTableProviderDelegateMock` instead `OrdersListTableProviderMock`. It will be more clearly to understand about mock's role.